lib/danger/request_sources/gitlab.rb

```diff
@@ -50,7 +50,7 @@ def validates_as_ci?
         includes_port = self.host.include? ":"
         raise "Port number included in `DANGER_GITLAB_HOST`, this will fail with GitLab CI Runners" if includes_port
 
-        super
+        GitLabCI.validates_as_ci?(environment)
```

Comment:
Good point, it would be good to add tests to catch this.

Reply:
That is a very good point. Indeed that sounds like this might break it. So this is actually not trying to validate if it's CI or not, but if it's hosted on GitLab or not?

In that case it sounds reasonable to check the remote host from Git.

Reply:
Yeah, just confirming that this code is aiming for "should I comment to GitLab" 👍